data/pref.xml

```diff
@@ -133,6 +133,10 @@
       <value id="BILINEAR_MIPMAP" value="2" />
       <value id="TRILINEAR_MIPMAP" value="3" />
     </enum>
+    <enum id="AlphaRange">
+      <value id="PERCENTAGE" value="0" />
+      <value id="EIGHT_BIT" value="1" />
```

Comment:
This is a minor thing, but I'd switch the values: PERCENTAGE=1 and EIGHT_BIT (or raw) =0 (these values don't care too much, but sometimes in scripts we have to use them as they are, and we can see `0` as the "most basic representation" (raw bytes) and then any following number as special representations).

```suggestion
      <value id="EIGHT_BIT" value="0" />
      <value id="PERCENTAGE" value="1" />
```